src/dotnet-interactive-vscode/common/interactiveClient.ts

```diff
@@ -163,7 +165,8 @@ export class InteractiveClient {
                     case StandardOutputValueProducedType:
                         {
                             let disp = <DisplayEvent>eventEnvelope.event;
-                            let output = this.displayEventToCellOutput(disp);
+                            const stream = eventEnvelope.eventType === StandardErrorValueProducedType ? 'stderr' : 'stdout';
+                            let output = this.displayEventToCellOutput(disp, stream);
```

Comment:
We need this so we can tell VS Code whether its stdout or stderr
Also required when saving ipynb